ui/src/components/RemoteAccess/ObserverDialog.jsx

```diff
@@ -32,13 +29,15 @@ export class ObserverDialog extends React.Component {
     );
   }
 
-  accept() {
+  accept(evt) {
+    evt.preventDefault();
```

Comment:
Entering a name and submitting the form was triggering a full page navigation/reload.